src/app/navbar/navbar.component.scss

```diff
@@ -1,50 +1,94 @@
 @import "src/styles/variables";
 @import "src/styles/maps";
+@import "src/styles/mixins";
 
 .navbar {
-  width: 100%;
-  height: 80px;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  background-color: $extra-light;
-
-  &__logo {
-    margin-left: 95px;
-    display: flex;
-    align-items: baseline;
-    text-decoration: none;
-
-    &--logo-h1 {
-      @include headings(logo-h1);
-    }
-
-    &--logo-h2 {
-      @include headings(logo-h2);
-    }
-  }
-
-  &__links {
-    margin-right: 95px;
-    display: flex;
-    gap: 20px;
-    align-items: center;
-    text-decoration: none;
-
-    &--link-h3 {
-      @include headings(h3);
-      text-decoration: none;
-      transition: 0.2s ease-in-out;
-
-      &:hover {
-        color: $light-blue;
-      }
-    }
-
-    &--active {
-      color: $blue;
-    }
-  }
+	width: 100%;
+	height: 80px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	background-color: $extra-light;
+	box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.2);
+	position: relative;
+	z-index: 2;
+
+	&__logo {
+		margin-left: $layout-spacing-large;
+		display: flex;
+		align-items: baseline;
+		text-decoration: none;
+
+		&--logo-h1 {
+			@include headings(logo-h1);
+		}
+
+		&--logo-h2 {
+			@include headings(logo-h2);
+		}
+	}
+
+	&__links {
+		margin-right: $layout-spacing-large;
+		display: flex;
+		gap: 20px;
+		align-items: center;
+		text-decoration: none;
+
+		&--link-h3 {
+			@include headings(h3);
+			text-decoration: none;
+			transition: 0.2s ease-in-out;
+
+			&:hover {
+				color: $light-blue;
+			}
+		}
+
+		&--active {
+			color: $blue;
+		}
+	}
+
+	@include mobile {
+		&__logo {
+			margin-left: $layout-spacing-small;
+
+			&--logo-h1 {
+				@include headings(logo-h1);
+				font-size: 20px;
```

Comment:
Do you want to have a separate `headings-mobile` map so you don't have to add the `font-size`s each time?